### PR TITLE
Add more information about permissions

### LINKED ISSUE #45: (Linux) Missing `rmmod` suggestions in FAQ
Currently, the kernel driver conflict solution ([the Troubleshooting section](https://opentabletdriver.net/Wiki/FAQ/Linux#troubleshooting)) describes how to set up udev rules, and ["My cursor is going crazy"](https://opentabletdriver.net/Wiki/FAQ/General#emi-interference) only talks about EMI. It is also possible that the user just installed the driver, and therefore the old drivers are still loaded.

`sudo rmmod wacom; sudo rmmod hid_uclogic` fixed the issue for at least 1 user on PopOS! [Discord verification](https://discord.com/channels/615607687467761684/723399565780189234/1175650582715957342)

### PR DESCRIPTION
**Pull request by X9VoiD**
## Changes

- Repurpose `UdevRules.md` into a documentation of the permissions that OTD needs, and how to grant them.
- Resolve #45

### CODE REVIEW

File: _wiki/FAQ/Linux.md [hunk 1]
```diff
@@ -14,21 +14,17 @@ Also check out [Linux App Specific FAQ]({% link _wiki/FAQ/LinuxAppSpecific.md %}
...
+##### Another driver is found {#hid_uclogic}
 
 _Symptoms_
 
 ```
-Another tablet driver found: UC Logic
-```
-
-```
-ArgumentOutOfRangeException: Value range is [0, 15]. (Parameter 'value)
```
Comment (gonX) [inline _wiki/FAQ/Linux.md:26]: Is this actually no longer displayed for users? I would have to check with a uclogic tablet to see whether it still displays this, before I can merge this change.

**Review by gonX — CHANGES_REQUESTED** [review]
Very nice change. LGTM aside from that one extra line removal that I can check myself.

File: _wiki/FAQ/Linux.md (hunk 1 above)
Comment (X9VoiD) [reply _wiki/FAQ/Linux.md:26]: The tablet driver detected message is enough to cover this as well. Only time that this message occurs is when `hid_uclogic` is loaded, which makes this redundant.

**Review by gonX — APPROVED** [review]
LGTM

**Pull request merged**